app/Http/Controllers/BoulderController.php

```diff
@@ -99,11 +99,13 @@ public function update(Request $request, $id)
      * Remove the specified resource from storage.
      *
      * @param int $id
-     * @return \Illuminate\Http\Response
+     * @return \Illuminate\Http\JsonResponse
+     * @throws \Exception
      */
     public function destroy($id)
     {
-        //
+        $success = Auth::user()->boulders()->find($id)->delete();
+        return response()->json($success);
```

Comment:
It's quite common to not return anything after a delete. So basically just a 204 response without body.
You can achieve that with `return response()->noContent();`

Reply:
ah that also is cool!